src/main/io/osd.c

```diff
@@ -645,9 +645,9 @@ static void osdDrawElements(void)
       return;
 
 #ifdef CMS
-    else if (sensors(SENSOR_ACC) || displayIsGrabbed(osdDisplayPort))
```

Comment:
`displayIsGrabbed(osdDisplayPort)` is always `false` here as it is a condition of `osdDrawElements()` being called (https://github.com/betaflight/betaflight/blob/master/src/main/io/osd.c#L1101).